examples/docker-for-mac/my_test.sh

```diff
@@ -21,15 +21,24 @@ set -o pipefail
 readonly IMAGE_TAG=v2.2.0
 readonly IMAGE_REPOSITORY="quay.io/helmpack/chart-testing"
 
-main() {
-    local testcontainer_id
-    testcontainer_id=$(create_testcontainer)
+run_ct_container() {
+    echo 'Running ct container...'
+    docker container run --rm --interactive --detach --name ct \
+        --volume "$(pwd):/workdir" \
+        --workdir /workdir \
+        "$IMAGE_REPOSITORY:$IMAGE_TAG"
+    echo
+}
 
-    # shellcheck disable=SC2064
-    trap "docker container rm --force $testcontainer_id > /dev/null" EXIT
+cleanup() {
+    echo 'Removing ct container...'
+    docker kill ct > /dev/null 2>&1
+
+    echo 'Done!'
+}
 
-    configure_kubectl "$testcontainer_id"
-    run_test
+docker_exec() {
+    docker exec --interactive -e HELM_HOST=127.0.0.1:44134 -e HELM_TILLER_SILENT=true ct "$@"
```

Comment:
Why do you add `HELM_HOST` and `HELM_TILLER_SILENT`. This sounds like tillerless Helm which we don't do here.

Reply:
If we do want to support people using for testing tillerless helm, we could do that easily with a [bash param substitution default](https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion) to false (`-e HELM_TILLER_SILENT=${HELM_TILLER_SILENT:-false}`), and then those who want can `export HELM_TILLER_SILENT=true` before running the script? Either way perhaps this should be a separate PR so we can discuss whether and how to support that option.

Reply:
After all, it's just an example. We don't have to cater for every possible use case here.